scripts/generate-app-config/generateEvidenceConfig.js

```diff
@@ -0,0 +1,118 @@
+const path = require('path');
+const glob = require('glob');
+const kebabCase = require('lodash.kebabcase');
+const {
+  startCase,
+  relativePath,
+  getNamespace,
+  parseExtension,
+} = require('./configHelpers');
+
+const getScreenshotPatterns = generatePagesOptions => (
+  generatePagesOptions.searchPatterns.reduce((acc, { root }) => {
+    const rootPath = root.replace(/[\\]/g, '/');
+    acc.push({
```

Comment:
I think these could be simplified to 
`pattern: `${rootPath}/packages/*/{test|tests}/wdio/__snapshots__/reference/**/*.png`,`

Reply:
Updated.